flutter_local_notifications_windows/lib/src/plugin/ffi.dart

```diff
@@ -22,10 +24,28 @@ extension on String {
       this[23] == '-';
 }
 
+/// Does a basic syntax check on XML.
+bool _checkXml(String xml) {
```

Comment:
It's file private but look this could have been private within the `FlutterLocalNotificationsWindows` class. Would you be able to adjust this?

Note: on a related note and not that it has to be done in this PR but I believe I missed picking this up when it comes to `_globalLaunchCallback` as well